src/openforms/tests/e2e/test_input_validation.py

```diff
@@ -292,3 +317,223 @@ def test_max_date_fixed_value(self):
+            "validate": {"required": True},
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="",
+            expected_ui_error="Het verplichte veld Required map is niet ingevuld.",
+        )
+
+
+class SinglePostcodeTests(ValidationsTestCase):
+    def test_required_field(self):
+        component: Component = {
+            "type": "postcode",
+            "key": "requiredPostcode",
+            "label": "Required postcode",
+            "validate": {"required": True},
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="",
+            expected_ui_error="Het verplichte veld Required postcode is niet ingevuld.",
+        )
+
+
+class SingleSignatureTests(ValidationsTestCase):
+
+    async def apply_ui_input(self, page: Page, label: str, ui_input: str | int | float):
+        page.wait_for_selector(f"[aria-label='{label}']", state="visible")
+
+    def test_required_field(self):
+        component: Component = {
+            "type": "signature",
+            "key": "requiredSignature",
+            "label": "Required signature",
+            "validate": {"required": True},
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="",
+            expected_ui_error="Het verplichte veld Required signature is niet ingevuld.",
+        )
+
+
+class SingleTextAreaTests(ValidationsTestCase):
+    def test_required_field(self):
+        component: Component = {
+            "type": "textarea",
+            "key": "requiredTextarea",
+            "label": "Required textarea",
+            "validate": {"required": True},
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="",
+            expected_ui_error="Het verplichte veld Required textarea is niet ingevuld.",
+        )
+
+    def test_max_length(self):
+        component: Component = {
+            "type": "textarea",
+            "key": "requiredTextarea",
+            "label": "Required textarea",
+            "validate": {"maxLength": 10},
+        }
+        invalid_sample = "word" * 4
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input=invalid_sample,
+            expected_ui_error="Er zijn teveel karakters opgegeven.",
+        )
+
+
+class SingleTimeTests(ValidationsTestCase):
+    def test_required_field(self):
+        component: Component = {
+            "type": "time",
+            "key": "requiredTime",
+            "label": "Required time",
+            "validate": {"required": True},
+        }
+
+        self.assertValidationIsAligned(
+            component,
+            ui_input="",
+            expected_ui_error="Het verplichte veld Required time is niet ingevuld.",
+        )
+
+    def test_min_value(self):
+        component: Component = {
+            "type": "time",
+            "key": "requiredTime",
+            "label": "Required time",
```

Comment:
it's nitpicking, but can you make the `key` and `label` have values that reflect back to the test itself? This component is not about the "required" validation, but about the min value. :)